### PR TITLE
Exposing ToggleTouchIdEnrollment through IOSDriver

### PR DESCRIPTION
**Pull request by StoneTree1**
## Change list

Adding ToggleTouchIdEnrollment to IOSDriver
 
## Types of changes

What types of changes are you proposing/introducing to .NET client?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [* ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Details

Please provide more details about changes if it is necessary. If there are new features you can provide code samples which show the way they
work and possible use cases. Also you can create [gists](https://gist.github.com) with pasted C# code samples or put them here using markdown. 
About markdown please read [Mastering markdown](https://guides.github.com/features/mastering-markdown/) and [Writing on GitHub](https://help.github.com/categories/writing-on-github/)

### CODE REVIEW

File: appium-dotnet-driver/Appium/AppiumCommand.cs [hunk 1]
```diff
@@ -58,8 +58,8 @@ public class AppiumCommand
                 new AppiumCommand(CommandInfo.PostCommand, AppiumDriverCommand.StartActivity, "/session/{sessionId}/appium/device/start_activity"),
                 new AppiumCommand(CommandInfo.GetCommand,  AppiumDriverCommand.GetSettings, "/session/{sessionId}/appium/settings"),
                 new AppiumCommand(CommandInfo.PostCommand, AppiumDriverCommand.UpdateSettings, "/session/{sessionId}/appium/settings"),
-		new AppiumCommand(CommandInfo.PostCommand, AppiumDriverCommand.TouchID, "/session/{sessionId}/appium/simulator/touch_id"),
-		new AppiumCommand(CommandInfo.PostCommand, AppiumDriverCommand.ToggleTouchIdEnrollment, "/session/{sessionId}/appium/simulator/toggle_touch_id_enrollment"),
+		        new AppiumCommand(CommandInfo.PostCommand, AppiumDriverCommand.TouchID, "/session/{sessionId}/appium/simulator/touch_id"),
```
Comment (jlipps) [inline appium-dotnet-driver/Appium/AppiumCommand.cs]: looks like spacing is off here

Comment (StoneTree1) [inline appium-dotnet-driver/Appium/AppiumCommand.cs]: Oh my email wasn't verified by Github when I made the commit. I have signed the CLA.
The spacing is odd, looks fine in visual studio. Maybe a spaces vs tabs issue?

**Comment by jlipps** [comment]
hi @StoneTree1 code looks good now. and you may have signed the CLA, but the code that was pushed to this PR belongs to an e-mail address that is not a github user. you can either close this and recreate the commits with the correct user, or do what the CLA bot is suggesting:

> Worley, Tristan (Auckland 1010) seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please add the email address used for this commit to your account.

**Comment by StoneTree1** [comment]
looks like source tree used wrong account when pushing. Sorting it now

**Comment by StoneTree1** [comment]
Hmm The commits are linked to my account now but still doesn't seem to accept the CLA. Guess i'll have to create a new pull request

**Comment by jlipps** [comment]
@StoneTree1 yes the CLA bot looks at the actual git commits (not the github user) and checks. So you will need to fixup the commits under the new user and re-push. I.e., if you do `git log` you should see that the commits are using the e-mail address which is signed up for github.

**Comment by StoneTree1** [comment]
CLA looks okay now

**Comment by jlipps** [comment]
thanks!

**Pull request merged**